src/net_processing.cpp

```diff
@@ -2005,18 +2005,29 @@ void PeerManagerImpl::SendPings()
 
 void PeerManagerImpl::RelayTransaction(const uint256& txid, const uint256& wtxid)
 {
-    LOCK(m_peer_mutex);
-    for(auto& it : m_peer_map) {
-        Peer& peer = *it.second;
-        auto tx_relay = peer.GetTxRelay();
-        if (!tx_relay) continue;
+    m_connman.ForEachNode([this, &wtxid, &txid](CNode* node) {
+        PeerRef peer{this->GetPeerRef(node->GetId())};
+        if (!peer) return false;
```

Comment:
```suggestion
        if (!peer) return;
```

nit: The function return type is `void`